src/stan/services/sample/init_softabs.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef STAN_SERVICES_SAMPLE_INIT_SOFTABS_HPP
```

Comment:
We can — but it hasn’t been exposed to CmdStan yet.
I don’t want to further slow down compiles.

On Apr 26, 2016, at 9:54 PM, Bob Carpenter <notifications@github.com> wrote:

> In src/stan/services/sample/init_softabs.hpp:
> 
> > @@ -0,0 +1,33 @@
> > +#ifndef STAN_SERVICES_SAMPLE_INIT_SOFTABS_HPP
> Does this mean we'll be able to run RHMC in CmdStan?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>